Fitness/Views/ClassDetail/ClassDetailDescriptionCell.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ClassDetailDescriptionCell.swift
+//  Fitness
+//
+//  Created by Kevin Chan on 5/18/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class ClassDetailDescriptionCell: UICollectionViewCell {
+
+    // MARK: - Private view vars
+    private let descriptionTextView = UITextView()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    // MARK: - Public configure
+    func configure(for gymClassInstance: GymClassInstance) {
+        descriptionTextView.text = gymClassInstance.classDescription
+    }
+
+    // MARK: - Private helpers
+    private func setupViews() {
+        descriptionTextView.font = ._14MontserratLight
+        descriptionTextView.isEditable = false
+        descriptionTextView.textAlignment = .center
+        descriptionTextView.sizeToFit()
```

Comment:
is this necessary if ur already setting constraints elsewhere?

Reply:
not my code but ill fix